djhtml/options.py

```diff
@@ -68,15 +68,15 @@
 
 # Parse arguments and assign attributes to self
 self = sys.modules[__name__]
-args = parser.parse_args(namespace=self)
+options = parser.parse_args()
```

Comment:
Previously, the `namespace=self` argument would assign the parsed options the module itself. This allows one to do:

```python
from djthml import options
options.name_of_option
```

I like the elegance and simplicity of this, but mypy doesn't, so the interface has now been changed slightly to:

```
from djthml.options import options
options.name_of_option
```